libraries/template.rb

```diff
@@ -0,0 +1,22 @@
+module Resolver
+  module Cookbook
+    module TemplateHelpers
+      def nil_or_empty?(property)
+        return true if property.nil? || (property.respond_to?(:empty?) && property.empty?)
+
+        false
+      end
```

Comment:
Would it make sense to return the conditional result directly?
```suggestion
      def nil_or_empty?(property)
        property.nil? || (property.respond_to?(:empty?) && property.empty?)
      end
```